tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -285,6 +285,16 @@ module.exports = {
         }
       }
     },
+    assertSharingWithCollaboratorIsNotPossible: function (collaborator, resource) {
+      return this.api.page
+        .FilesPageElement
+        .filesList()
+        .closeSidebar(100)
+        .openSharingDialog(resource)
+        .enterAutoComplete(collaborator)
```

Comment:
I think It's a good idea to wait for some ajax calls here as the dropdown may not be visible immediately.